api/v1alpha1/oidcpolicy_types.go

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
Can we get these files to be under `cmd/extension/` or somewhere _specific_? Wondering if we could get all files related to a single extension under one single directory?

Reply:
It's not quite possible without adding specific targets to use the build tools from kustomize/operator-sdk